portfolio/src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.CalendarEvent;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles calendar event creation. */
+@WebServlet("/calendarServlet")
+public class CalendarServlet extends HttpServlet {
+
+  public List<CalendarEvent> events;
+
+  public CalendarServlet() {
+    events = new ArrayList<CalendarEvent>();
+  }
+
+  /**
+   * Retrieves the new calendar event's information from the POST request. Creates a new instance of
+   * the CalendarEvent class and adds it to a list of calendar events.
```

Comment:
I still see that this servlet is stateful. For example, consider the scenario below.
1. User A visits the web app, and invokes this servlet method with their (private) calendar events.
2. User B does the same with their own private calendar events.

Then this servlet will end up storing both users' calendar events, right? So I think it'd be simpler/safer to make the servlet stateless.

For example, consider introducing ScheduleTasksServlet, which takes in the user's tasks (to complete) and existing events, as well as preferences etc, returns the scheduling output. It does not have to update some local variable.

Reply:
I see. I originally built the Calendar servlet thinking that there will be some backend functionalities that only apply to the calendar part, but based on the goal of making the servlet stateless, I don't think it's necessary to have a separate servlet for the calendar anymore. I think the servlet that handles taking in the tasks and events and outputting the schedule is something Tomas is building already, so in this PR, I will delete the unnecessary CalendarServlet.java file for now. If it turns out that Tomas had other plans of incorporating the scheduling algorithm with a servlet, I will revisit this with him. 

As of right now, the calendar's client-side only needs to send a list of calendar events to the ScheduleTaskServlet, so having an additional CalendarServlet is not necessary.